rivet.spec

```diff
@@ -11,44 +11,26 @@ Patch0: rivet-1.4.0
 %endif
 
 %if "%{?cms_cxxflags:set}" != "set"
-%define cms_cxxflags -O2 -std=c++11
+%define cms_cxxflags -O2 -std=c++0x
 %endif
 
 %prep
 %setup -n rivet/%{realversion}
 %patch0 -p0
 
-# Update config.{guess,sub} to detect aarch64 and ppc64le
-rm -f %{_tmppath}/config.{sub,guess}
-curl -L -k -s -o %{_tmppath}/config.guess 'http://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.guess;hb=HEAD'
-curl -L -k -s -o %{_tmppath}/config.sub 'http://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.sub;hb=HEAD'
-for CONFIG_GUESS_FILE in $(find $RPM_BUILD_DIR -name 'config.guess')
-do
-  rm -f $CONFIG_GUESS_FILE
-  cp %{_tmppath}/config.guess $CONFIG_GUESS_FILE
-  chmod +x $CONFIG_GUESS_FILE
-done
-for CONFIG_SUB_FILE in $(find $RPM_BUILD_DIR -name 'config.sub')
-do
-  rm -f $CONFIG_SUB_FILE
-  cp %{_tmppath}/config.sub $CONFIG_SUB_FILE
-  chmod +x $CONFIG_SUB_FILE
-done
```

Comment:
Keep this unless you know that all `config.{guess,sub}` scripts are up to date and can work on _aarch64_ and _ppc64le_ architectures. You can also ping upstream to get these updated to newer ones.